sklearn/model_selection/_split.py

```diff
@@ -920,7 +923,6 @@ def get_n_splits(self, X, y, groups):
         if groups is None:
             raise ValueError("The groups parameter should not be None")
```

Comment:
Can you change too:
`raise ValueError("The 'groups' parameter should not be None.")`